AABB_tree/doc/AABB_tree/PackageDescription.txt

```diff
@@ -5,12 +5,12 @@
 
 /*!
 \addtogroup PkgAABBTreeRef
-\cgalPkgDescriptionBegin{3D Fast Intersection and Distance Computation,PkgAABBTree}
+\cgalPkgDescriptionBegin{2D and 3D Fast Intersection and Distance Computation,PkgAABBTree}
```

Comment:
```suggestion
\cgalPkgDescriptionBegin{Fast Intersection and Distance Computation,PkgAABBTree}
```

Reply:
WIth or without `2D and 3D` seems fine. The AABBTree should be able to handle d dimensions.